### PR TITLE
[FP-2185] Add Support for Google Ads Enhanced Conversions

### PR DESCRIPTION
**Pull request by mtran999**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -719,13 +756,6 @@ ___TEMPLATE_PARAMETERS___
         "checkboxText": "Include user-provided data from your website",
         "simpleValueType": true,
         "help": "User-provided data allows you to improve the accuracy of your measurement by sending hashed first party user-provided data from your website. You will need to agree to the user-provided data terms and policies in your Search Ads 360 account first.",
-        "enablingConditions": [
-          {
-            "paramName": "tagType",
-            "type": "EQUALS",
-            "paramValue": "floodlightEvent"
-          }
-        ]
```
Comment (bianchidotdev) [inline template.tpl:728]: Nice

File: template.tpl [hunk 2]
```diff
@@ -342,6 +342,43 @@ ___TEMPLATE_PARAMETERS___
...
+      {
+        "type": "CHECKBOX",
+        "name": "googleAdsEnhancedConversionsCheckbox",
+        "checkboxText": "Include user-provided data from your website",
+        "simpleValueType": true,
+        "help": "Improve measurement and get more insights with data people provide to your website. You will need to agree to the user-provided data terms and policies in your Google Ads account first.",
+      },
+      {
+        "type": "SELECT",
+        "name": "googleAdsEnhancedConversionsUserDataVariable",
```
Comment (bianchidotdev) [inline template.tpl:367]: It might be good to include helptext here saying that the variable selected should be of type ` User-Provided Data`

**Review by bianchidotdev — APPROVED** [review]
Very cool. Just one nitpick

File: template.tpl (hunk 2 above)
Comment (mtran999) [reply template.tpl:367]: Good idea. I just copy pasted the original help text lol

**Pull request merged**